solutions/usaco-491.mdx

```diff
@@ -10,6 +10,79 @@ author: Qi Wang
 **Time Complexity:** $\mathcal{O}(N)$
 
 <LanguageSection>
+	
+<CPPSection>
+
+```cpp
+
+#include <bits/stdc++.h>
+using namespace std;
+#define fastio ios_base::sync_with_stdio(false); cin.tie(NULL);
```

Comment:
don't think this is part of the allowed macros

Reply:
I have removed that from the code